grid-client/node/node.go

```diff
@@ -374,6 +374,34 @@ func (n *NodeClient) SystemVersion(ctx context.Context) (ver Version, err error)
 	return
 }
 
+// GetPerfTestsResults get all perf tests results
+func (n *NodeClient) GetPerfTestsResults(ctx context.Context) (result string, err error) {
```

Comment:
While I don't like naked returns, but I understand you prefered to be consistent with the rest of the module